### ISSUE TITLE
fix cstyle error in src/config.h

### ISSUE THREAD

**Issue by ashishranjan738** [openebs/openebs#2009]
Repo: openebs/istgt
Branch: replication
File: src/config.h
Steps to compile in: README.md
PR subject should be like: fmt(cstyle): cstyle fixes in <filename>
```
./src/config.h: 8: #define followed by space instead of tab
./src/config.h: 11: #define followed by space instead of tab
./src/config.h: 14: #define followed by space instead of tab
./src/config.h: 17: #define followed by space instead of tab
./src/config.h: 23: #define followed by space instead of tab
./src/config.h: 38: #define followed by space instead of tab
./src/config.h: 41: #define followed by space instead of tab
./src/config.h: 44: #define followed by space instead of tab
./src/config.h: 47: #define followed by space instead of tab
./src/config.h: 50: #define followed by space instead of tab
./src/config.h: 53: #define followed by space instead of tab
./src/config.h: 56: #define followed by space instead of tab
./src/config.h: 59: #define followed by space instead of tab
./src/config.h: 62: #define followed by space instead of tab
./src/config.h: 67: improper first line of block comment
./src/config.h: 68: indent by spaces instead of tabs
./src/config.h: 69: #define followed by space instead of tab
./src/config.h: 75: #define followed by space instead of tab
./src/config.h: 78: #define followed by space instead of tab
./src/config.h: 81: #define followed by space instead of tab
./src/config.h: 84: #define followed by space instead of tab
./src/config.h: 87: #define followed by space instead of tab
./src/config.h: 90: #define followed by space instead of tab
./src/config.h: 99: #define followed by space instead of tab
./src/config.h: 101: improper first line of block comment
./src/config.h: 102: indent by spaces instead of tabs
./src/config.h: 103: #define followed by space instead of tab
./src/config.h: 106: #define followed by space instead of tab
./src/config.h: 109: #define followed by space instead of tab
./src/config.h: 112: #define followed by space instead of tab
./src/config.h: 118: #define followed by space instead of tab
./src/config.h: 124: #define followed by space instead of tab
./src/config.h: 127: #define followed by space instead of tab
./src/config.h: 130: #define followed by space instead of tab
./src/config.h: 133: #define followed by space instead of tab
./src/config.h: 136: #define followed by space instead of tab
./src/config.h: 139: #define followed by space instead of tab
./src/config.h: 142: #define followed by space instead of tab
./src/config.h: 148: #define followed by space instead of tab
./src/config.h: 151: #define followed by space instead of tab
./src/config.h: 154: #define followed by space instead of tab
./src/config.h: 157: #define followed by space instead of tab
./src/config.h: 160: #define followed by space instead of tab
./src/config.h: 162: improper first line of block comment
./src/config.h: 163: indent by spaces instead of tabs
./src/config.h: 164: #define followed by space instead of tab
./src/config.h: 167: #define followed by space instead of tab
./src/config.h: 179: #define followed by space instead of tab
./src/config.h: 182: #define followed by space instead of tab
./src/config.h: 185: #define followed by space instead of tab
./src/config.h: 188: #define followed by space instead of tab
./src/config.h: 191: #define followed by space instead of tab
./src/config.h: 194: #define followed by space instead of tab
./src/config.h: 197: #define followed by space instead of tab
./src/config.h: 211: improper first line of block comment
./src/config.h: 212: indent by spaces instead of tabs
./src/config.h: 213: #define followed by space instead of tab
./src/config.h: 216: #define followed by space instead of tab
./src/config.h: 219: #define followed by space instead of tab
./src/config.h: 222: #define followed by space instead of tab
./src/config.h: 225: #define followed by space instead of tab
./src/config.h: 228: #define followed by space instead of tab
./src/config.h: 231: #define followed by space instead of tab
./src/config.h: 234: #define followed by space instead of tab
./src/config.h: 237: #define followed by space instead of tab
./src/config.h: 246: #define followed by space instead of tab
./src/config.h: 253: blank after preprocessor #
./src/config.h: 260: #define followed by space instead of tab
./src/config.h: 265: improper first line of block comment
./src/config.h: 265: missing space around relational operator
./src/config.h: 266: indent by spaces instead of tabs
./src/config.h: 266: missing space around relational operator
./src/config.h: 267: indent by spaces instead of tabs
./src/config.h: 267: preprocessor statement not in column 1
./src/config.h: 270: improper first line of block comment
./src/config.h: 270: missing space around relational operator
./src/config.h: 271: indent by spaces instead of tabs
./src/config.h: 271: missing space around relational operator
./src/config.h: 272: indent by spaces instead of tabs
./src/config.h: 272: preprocessor statement not in column 1
./src/config.h: 275: improper first line of block comment
./src/config.h: 275: missing space around relational operator
./src/config.h: 276: indent by spaces instead of tabs
./src/config.h: 276: missing space around relational operator
./src/config.h: 277: indent by spaces instead of tabs
./src/config.h: 277: preprocessor statement not in column 1
./src/config.h: 280: improper first line of block comment
./src/config.h: 281: indent by spaces instead of tabs
./src/config.h: 286: improper first line of block comment
./src/config.h: 287: indent by spaces instead of tabs
./src/config.h: 290: improper first line of block comment
./src/config.h: 291: indent by spaces instead of tabs
./src/config.h: 312: improper first line of block comment
./src/config.h: 313: indent by spaces instead of tabs
./src/config.h: 316: improper first line of block comment
./src/config.h: 317: indent by spaces instead of tabs
./src/config.h: 320: improper first line of block comment
./src/config.h: 321: indent by spaces instead of tabs
./src/config.h: 324: improper first line of block comment
./src/config.h: 325: indent by spaces instead of tabs
```
Resources:
-> Countributers guide: https://github.com/openebs/openebs/blob/master/CONTRIBUTING.md

Complexity: Easy

**Comment by danimart1991** [comment]
I take this. 😊

**Comment by marcoandredinis** [comment]
I'm not sure we should _fix_ the style in this file, as it is auto-generated.

The error `#define followed by space instead of tab` comes from the script `./configure` @ line 6216
`print prefix "define", macro P[macro] D[macro]`
If we replace this line with `print prefix "define\t" macro P[macro] D[macro]` we get rid of those errors.
However, this file (`configure`) is also auto-generated.
I did not dig into the remaining errors, but their cause should be the same.

So, instead of trying to fix this style error, I've opened a PR which disables style check for this file (https://github.com/openebs/istgt/pull/116)
